core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -131,6 +147,11 @@ public static ServerBuilder builder() {
 
         setupVersionMetrics();
 
+        if (config().sslContextMapping() != null) {
+            final SslContext sslContext = config().sslContextMapping().map(defaultHostname());
+            setupTlsMetrics(sslContext);
```

Comment:
Should we set up metrics for all `SslContext`s? We can add the hostname as a tag.

Reply:
I'll iterate over `config.virtualHosts` and set metric per each, is that sounds good?

Reply:
It sounds good.